src/App.js

```diff
@@ -1,23 +1,45 @@
-import logo from './logo.svg';
 import './App.css';
+import {useState} from 'react'
+import {Route, Switch} from "react-router-dom"
+import Home from "./pages/Home"
+import Login from "./pages/Login"
+import Garden from "./pages/MyGarden"
+import Calendar from "./pages/Calendar"
+import Profile from "./pages/MyProfile"
+import Team from "./pages/Team"
+import Plant from "./pages/Plant"
+import Nav from "./components/Nav"
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Switch>
+        <Route exact path="/">
+          <Home/>
+        </Route>
+        <Route path="/login">
+          <Login/>
+        </Route>
+        <Route path="/garden">
+          <Garden/>
+        </Route>
+        <Route path="/plant/:id"
```

Comment:
What if we made the path for the single plant page "/plant/name"? We'll have the given name of the plant provided by the user, and it would be a little more descriptive in the url address bar. We don't want to use the ObjectId here, and that could also help prevent confusion between frontend and backend routes.